addons/postgresql/templates/componentdefinition-12.yaml

```diff
@@ -147,6 +149,11 @@ spec:
             - -c
           args:
             - curl -s http://$(KB_REPLICATION_PRIMARY_POD_FQDN):8008/switchover -XPOST -d '{"leader":"$(KB_REPLICATION_PRIMARY_POD_NAME)"}'
+    preTerminate:
+      customHandler:
+        exec:
+          image: {{ .Values.etcdCleanToolsImage.registry | default "docker.io" }}/{{ .Values.etcdCleanToolsImage.repository }}:{{ .Values.etcdCleanToolsImage.tags }}
```

Comment:
Does it work? I remember the controller enforces all Actions must use the same image in the code, since they're executed by the kbagent (previous, lorry) container. And the image of kbagent is determined by `exec.image`.

Reply:
yes it works, The preterminate action involves launching another pod without using Lorry.

Reply:
The most recent KB version has replaced lorry to kbagent, which checks exec.image of all actions.